config/content.d/developer.rackspace.com.json

```diff
@@ -23,7 +23,7 @@
           "/docs/cloud-networks/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-networks/",
           "/docs/cloud-big-data/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-big-data/v2/",
           "/docs/autoscale/v1/developer-guide/": "https://github.com/rackerlabs/otter/",
-          "/docs/cloud-servers/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-servers/",
+          "/docs/cloud-servers/v2/": "https://github.com/rackerlabs/docs-cloud-servers/v2/",
           "/docs/cloud-files/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-files/",
```

Comment:
The contentID has to match the ID in the [_deconst.json](https://github.com/rackerlabs/docs-cloud-servers/blob/master/api-docs/_deconst.json) file: `` "contentIDBase": "https://github.com/rackerlabs/docs-cloud-servers/",``

You'll need to update the ``_deconst.json`` file, or remove the v2 here.  You don't really need the v2 because the content gets deployed to the v2 URL.   If you had multiple projects in the repo, then you'd want to add the version to the ID to differentiate one from the other.